ruby/overrides.js

```diff
@@ -19,5 +19,6 @@
     'should return browserInfo in getAllTestResults': {skip: true},
     'should be empty if page delayed by 1500': {skip: true},
 
-    "Should return exception in TestResultsSummary": {skipEmit: true},
+    'Should return exception in TestResultsSummary': {skipEmit: true},
+    'should get dom pseudo elements when check window': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9515763)
```suggestion
    "should get dom pseudo elements when check window": {skipEmit: true},
```